tests/bsim/bluetooth/audio/src/bap_broadcast_source_test.c

```diff
@@ -297,10 +425,6 @@ static int setup_extended_adv(struct bt_bap_broadcast_source *source, struct bt_
 
 static void test_broadcast_source_reconfig(struct bt_bap_broadcast_source *source)
 {
-	uint8_t bis_codec_data[] = {
-		BT_AUDIO_CODEC_DATA(BT_AUDIO_CODEC_CFG_FREQ,
-				    BT_BYTES_LIST_LE16(BT_AUDIO_CODEC_CFG_FREQ_16KHZ)),
-	};
```

Comment:
It is imperative that you set BIS specific data. 

These should definitely not be removed.

Reply:
BIS specific data is moved as global variable